src/runner.js

```diff
@@ -1,22 +1,24 @@
 const {
-  getPackageInfoList, writeReportFile, extractInvalidPackages, formatForbiddenLicenseError
+  getPackageInfoList, extractInvalidPackages, formatForbiddenLicenseError
 } = require('./utils');
 
+const reporter = require('./reporter');
+
 const run = async (checker, args) => {
   const packages = await checker.parsePackages(args.start);
 
   const packageList = getPackageInfoList(packages);
 
   const forbiddenPackages = extractInvalidPackages(args.failOn, packageList);
   if (forbiddenPackages.length) {
-    writeReportFile(args.errorReportFileName, forbiddenPackages, args.customHeader);
+    reporter.writeErrorReportFile(args.errorReportFileName, forbiddenPackages);
```

Comment:
As you said in the PR description, in the future we can add a property to customize the header for the error report